app/styles/teachers/teacher-student-view.sass

```diff
@@ -0,0 +1,120 @@
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
+@import "app/styles/style-flat-variables"
+
+#teacher-student-view
+  .breadcrumbs
+    color: #065e73
+    a
+      color: #065e73
+
+  .student-details
+    margin-top: 20px
+
+  .playtime-charts
+    margin-top: 30px
+
+    h4
+      margin-bottom: 15px
+      // font-weight: 600
+
+
+  svg[id|="visualisation"]
```

Comment:
Added a class - was originally keying off the IDs that I already needed for the graph, but  like you said, using classes makes more sense.